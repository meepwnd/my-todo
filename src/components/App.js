import React from 'react';
import AddOption from './AddOption';
import Options from './Options';
import RemoveButton from './RemoveButton';
import VisibilityFilters from './VisibilityFilters';
import '../styles/main.css';
import ClearCompletedButton from './ClearCompleted';
import ItemsCount from './ItemsCount'
import CompleteAllButton from './CompleteAllButton'

export default class App extends React.Component {
	render() {
		return (
			<div className="container">
				<h1 className="heading">Indesicion App</h1>
				<RemoveButton />
				<ClearCompletedButton />
				<AddOption />
				<Options />
				<ItemsCount />
				<VisibilityFilters />
				<CompleteAllButton />
			</div>
		)
	}
}

