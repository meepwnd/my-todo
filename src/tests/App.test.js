import React from 'react';
import { shallow } from 'enzyme';
import App from '../components/App';

it('renders welcome message', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});