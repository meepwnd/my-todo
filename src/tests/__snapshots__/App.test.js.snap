// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders welcome message 1`] = `
<div
  className="container"
>
  <h1
    className="heading"
  >
    Indesicion App
  </h1>
  <Connect(RemoveButton) />
  <Connect(AddOption) />
  <Connect(Options) />
  <Connect(VisibilityFilters) />
</div>
`;
